src/install_update.ts

```diff
@@ -79,12 +87,18 @@ export async function createUpdateResp(
         );
         updateResp.updates = [...updateResp.updates, ...fileUpdateResp];
       } catch (err) {
+        hookCLI.error(
+          "ohnoes install-update errored during updating of dependency files!",
```

Comment:
Same question about additional log line

Reply:
Same answer! No specific reason other than testing stuff out. Happy to remove it, lemme know.